elements/bulbs-video/components/revealed.js

```diff
@@ -256,11 +260,16 @@ export default class Revealed extends React.Component {
     GoogleAnalytics.init(this.player, videoMeta.gaTrackerAction);
     Comscore.init(this.player, global.BULBS_ELEMENTS_COMSCORE_ID, videoMeta.player_options.comscore.metadata);
 
+    this.player.on('complete', this.forwardJWEvent);
+  }
+
+  forwardJWEvent (event) {
+    this.refs.videoViewport.dispatchEvent(new CustomEvent(`jw-${event.type}`));
   }
 
   render () {
     return (
-      <div className='bulbs-video-viewport'>
+      <div className='bulbs-video-viewport' ref='videoViewport'>
```

Comment:
jwplayer seems to discard/replace the element you tell it to use, so I needed a hook to this wrapping element to dispatch events from.